Xamarin.PropertyEditing.Tests/Xamarin.PropertyEditing.Tests.csproj

```diff
@@ -19,6 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="Moq" Version="4.10.1" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
```

Comment:
For reference, `Moq` requires System.Threading.Tasks.Extensions  4.5.1 or newer, so i just upgraded to the latest stable and it started working in VSMac's test runner.